escape-chars.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Escape all unescaped '>' characters except for the lines starting with "title:"
+for file in `find ./docs/api`; do
+    if [ -f "$file" ]; then
+        mv "$file" "$file.back"
+        sed -e '/^title:/n;s/\([^\\]\)\>/\1\\\>/g' "$file.back" &> "$file"
+        if [ $? -eq 0 ]; then
+            rm "$file.back"
+            echo "fixed: $file"
+        else
+            mv "$file.back" "$file"
+            echo "error: $file"
+        fi
+    fi
+done
```

Comment:
Can you add info this file exists to the README?

Reply:
Thought I added, I guess I didn't.

Reply:
I actually did ;-) 
https://github.com/ethersphere/bee-js-docs/pull/54/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R74